### PR TITLE
support specifying the version with the package name

### PR DESCRIPTION
**Pull request by hectorj2f**
Allow specifying a single package version from multiple ones:

```
packages:
    - name: rustix@0.37.7
      version: 0.37.25
    - name: rmp-serde
      version: 1.1.1
```

### CODE REVIEW

**Comment by hectorj2f** [comment]
```
cargobump --bump-file=/Users/hectorfernandez/os/zellij/cargobump-deps.yaml 
2024/11/21 12:28:23 INFO Update package: rmp-serde
2024/11/21 12:28:28 INFO Package updated successfully: rmp-serde to version 1.1.1
2024/11/21 12:28:28 INFO Update package with a specific version: rustix@0.37.7
2024/11/21 12:28:29 INFO Package updated successfully: rustix@0.37.7 to version 0.37.25
```

File: pkg/update.go [hunk 1]
```diff
@@ -2,11 +2,10 @@ package pkg
 
 import (
 	"fmt"
-	"log"
-
 	"github.com/chainguard-dev/cargobump/pkg/run"
 	"github.com/chainguard-dev/cargobump/pkg/types"
 	"golang.org/x/mod/semver"
+	"log"
```
Comment (cpanato) [inline pkg/update.go:8]: why this change?

**Review by cpanato — APPROVED** [review]

File: pkg/update.go (hunk 1 above)
Comment (hectorj2f) [reply pkg/update.go:8]: my editor

**Review by rawlingsj — APPROVED** [review]
